# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
# SPDX-License-Identifier: BSD-3-Clause-Clear
image_width: 1920
image_height: 1080
camera_name: ar0231
camera_matrix:
  rows: 3
  cols: 3
  data: [   720    ,     0.    ,   960    ,
             0.    ,    720    ,   540    ,
             0.    ,     0.    ,    1.    ]
distortion_model: equidistant
distortion_coefficients:
  rows: 1
  cols: 4
  data: [    0.    ,    0.    ,    0.    ,    0.    ]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [   720     ,     0.    ,   960     ,    0.     ,
             0.     ,    720    ,   540     ,    0.     ,
             0.     ,     0.    ,    1.     ,    0.     ]
